data/persons.txt

```diff
@@ -944,23 +942,24 @@ person "Prototype B3-CC4"
 		word
 			">"
 	ship "Shooting Star" "Prototype B3-CC4"
-		"never disabled"
 
 ship "Shooting Star"
+	plural "Shootings Star"
+	noun "genocidal maniac"
 	sprite "ship/shooting star/shooting star"
 		"frame time" 3
-	"never disabled"
 	attributes
 		category "Heavy Warship"
 		"hull" 31500
```

Comment:
Maybe make this have a non-standard disabled threshold just to mess around. This would drop its threshold from 15% to 5%.
```suggestion
		"hull" 31500
		"threshold percentage" 0.05
```